uobtheatre/productions/test/test_schema.py

```diff
@@ -865,9 +873,9 @@ def test_performance_schema(gql_client):
                         "node": {
                             "createdAt": performance.created_at.isoformat(),
                             "updatedAt": performance.updated_at.isoformat(),
-                            "capacity": performance.capacity,
+                            "capacity": None,
```

Comment:
Why? Does this not depend on the implementation of the factory? (Same for disabled)